app/src/main/java/com/example/next_show/models/Show.java

```diff
@@ -201,6 +201,12 @@ public void setParseFields(ParseUser currUser) {
         setParseNetwork(network);
         setParseYearAired(year_aired);
         setParseUser(currUser);
+
+        // null check before adding to Parse
+        if(liked == null){
+            Log.i("Show Object", "No rating!");
+            liked = "";
+        }
```

Comment:
Is this expected or handling an error case?  If it's an error, I would use `Log.e`